lib/fathead/wikihow/output.txt

```diff
@@ -0,0 +1,15 @@
+How to "Bone" a Holster to Fit Your Handgun	A									[[Image:http://www.wikihow.com/images/a/a7/"Bone"-a-Holster-to-Fit-Your-Handgun-Step-10.jpg]]	<ol><li>Remove the contents from the packaging.</li><li>Obtain a suitably sized cooking pot from your cupboard and fill it with enough water to completely submerge the holster.</li><li>Prepare your handgun by applying plastic wrap over the entire weapon and use tape to secure it in place.</li><li>Apply the sight clearance tube over the front sight and align it along the center of the top to the rear sight and tape into place.</li><li>Apply another layer of plastic wrap and tape into place.</li><li>By now the water should be boiling.</li><li>Being careful of the heat of the holster, insert the handgun with the stitching centered on the trigger guard</li><li>Using the 'boning' tool, held at a 15-20 degree angle from the leather and applying firm pressure, begin to work the leather along the lines of your handgun.</li><li>Continue to work the leather to the handgun until you are satisfied with your creation.</li><li>Remove the plastic wrap and sight clearance tube.</li></ol>	http://www.wikihow.com/%22Bone%22-a-Holster-to-Fit-Your-Handgun
```

Comment:
@TiagoDanin it's actually not available in HTTPS, HTTPS on that link forwards to HTTP:

![untitled](https://user-images.githubusercontent.com/9503662/29467968-6664edc4-8410-11e7-86f3-1ce4095dbec4.png)